src/test/java/core/basesyntax/FruitServiceImplTests.java

```diff
@@ -0,0 +1,134 @@
+                new Fruit("apple"), 100));
+
+        fruitService.processData(fruitTransactions);
+        Map<Fruit, Integer> expected = Map.of(new Fruit("banana"), 120, new Fruit("apple"), 200);
+        Map<Fruit, Integer> actual = Storage.fruits;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void processData_oneFruitTransactionPurchase_ok() {
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.BALANCE,
+                new Fruit("banana"), 20));
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.BALANCE,
+                new Fruit("apple"), 100));
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.PURCHASE,
+                new Fruit("banana"), 10));
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.PURCHASE,
+                new Fruit("apple"), 20));
+
+        fruitService.processData(fruitTransactions);
+        Map<Fruit, Integer> expected = Map.of(new Fruit("banana"), 10, new Fruit("apple"), 80);
+        Map<Fruit, Integer> actual = Storage.fruits;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void processData_oneFruitTransactionReturn_ok() {
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.BALANCE,
+                new Fruit("banana"), 20));
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.BALANCE,
+                new Fruit("apple"), 100));
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.RETURN,
+                new Fruit("banana"), 20));
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.RETURN,
+                new Fruit("apple"), 50));
+
+        fruitService.processData(fruitTransactions);
+        Map<Fruit, Integer> expected = Map.of(new Fruit("banana"), 40, new Fruit("apple"), 150);
+        Map<Fruit, Integer> actual = Storage.fruits;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void processData_allFruitTransactions_ok() {
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.BALANCE,
+                new Fruit("banana"), 20));
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.BALANCE,
+                new Fruit("apple"), 100));
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.SUPPLY,
+                new Fruit("banana"), 100));
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.RETURN,
+                new Fruit("apple"), 10));
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.PURCHASE,
+                new Fruit("banana"), 5));
+
+        fruitService.processData(fruitTransactions);
+        Map<Fruit, Integer> expected = Map.of(new Fruit("banana"), 115, new Fruit("apple"), 110);
+        Map<Fruit, Integer> actual = Storage.fruits;
+        assertEquals(expected, actual);
```

Comment:
```suggestion
        assertEquals(expected, Storage.fruits);
```
we can also write this way